src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputMissingJavadocTypeIncorrect.java

```diff
@@ -34,4 +34,23 @@ class MyMethodClass {}
 
   class AdditionalClass { // OK, not public
   }
+
+  // violation below 'Missing a Javadoc comment.'
+  protected int testProtectedMethod1() {
+    return 0;
+  }
+
+  // violation below 'Missing a Javadoc comment.'
+  protected void testProtectedMethod2() {}
```

Comment:
please add private methods and fields and contructors
please add  methods and fields and constructorss wihtout visibility modifiers (aka package-private)

Reply:
I have added many new test cases, pls check and lemme know if we need more. Though for now I have only added it in InputIncorrect file, I will add it to InputCorrect file once we have a solid xpath query ready

Reply:
I have added public/protected class/interface/enum/record with public/protected/private/package-private methods and constructors in both input file: InputCorrect & InputIncorrect